x-pack/plugins/ml/server/models/data_recognizer/modules/metrics_ui_hosts/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "metrics_hosts",
+    "title": "Metrics Hosts",
+    "description": "",
+    "type": "Metricbeat Data",
+    "logoFile": "logo.json",
+    "defaultIndexPattern": "metricbeat-*",
+    "query": {
+        "bool": {
+          "must": [
+            {"exists": {"field": "system"}}
+          ],
+          "must_not": [
+            {"bool": {"filter": [{"exists":{"field": "kubernetes"}}]}},
+            {"bool": {"filter": [{"exists":{"field": "docker"}}]}}
+            ]
+        }
+      },
+    "jobs": [
+        {
+            "id": "hosts_cpu_usage",
```

Comment:
Noticed that the host jobs have an id prefixed with `hosts`, but the k8s jobs don't. Should we be consistent here? Not sure we actually need the prefix.

Reply:
i've included the k8s prefix on the kubernetes jobs, as we'll want to distinguish between them in ML job management.